content/en/network_monitoring/devices/network_topology_map.md

```diff
@@ -15,62 +15,209 @@ further_reading:
+
+### Empty map/ no monitored devices
+
+- Ensure the "Hide _N_ Unconnected Devices toggle is off.
+
+### Missing devices/connections
+
+The Device Topology Map data is based on LLDP (Link Layer Discovery Protocol) and CDP (Cisco Discovery Protocol) information collected with SNMP. If your map is missing devices and/or connections, ensure the following:
+
+- Datadog Agent version 7.52 or later is installed.
+- Devices have LLDP and/or CDP enabled with SNMP.
+
+Verify that your devices are exposing LLDP and CDP data with the following commands:
+
+For LLDP data:
 
 ```yaml
 sudo -u dd-agent datadog-agent snmp walk <DEVICE_IP> 1.0.8802
 ```
-
-```yaml
+For CDP data
+```yaml:
 sudo -u dd-agent datadog-agent snmp walk <DEVICE_IP> 1.3.6.1.4.1.9.9.23
 ```
 
+### Missing connections or links 
+
+If your device is exposing topology data with LLDP or CDP but some of the connections are missing, ensure that the "Hide _N_ Unmonitored Devices" toggle is off.
+If you are using tags to filter nodes on the map, ensure the "Show one hop away on filter" toggle is on to see the connected nodes.
+
+### Un-monitored devices showing on map
+
+The Device Topology Map shows all devices discovered with LLDP or CDP. These can be new devices that are not already monitored with SNMP or existing devices that were not [resolved](#device-resolution) to the equivalent monitored device.
+You can use the "Hide _N_ Unmonitored Devices" toggle to hide these nodes.
+
+### Device duplicated on map
+
+The Device Topology Map shows all devices discovered with LLDP and/or CDP. In some cases, these devices are already monitored with SNMP but can not be [resolved](#device-resolution) to the equivalent monitored device. In this case, the device is shown twice: one node representing the monitored device and one node representing the LLDP/CDP discovered device.
+Use the "Hide _N_ Unmonitored Devices" toggle to hide the unmonitored nodes.
+
+### Borderless or black nodes on the map 
+
+The borderless or black nodes on the Device Topology Map can represent devices discovered with LLDP or CDP that are not configured to be monitored with NDM, or devices discovered with LLDP or CDP that can not be resolved to the equivalent [monitored device](#device-resolution).
+
+## Device resolution 
+
+The Device Topology Map provides an overview of the devices monitored with NDM and their physical connections. The topology links data is based on LLDP (Link Layer Discovery Protocol) or CDP (Cisco Discovery Protocol) information collected with SNMP. 
+The connections discovered with LLDP or CDP can correspond to devices already monitored with SNMP. The device resolution consists in matching the discovered device to the monitored device.
+
+### Device resolution failures
+
+The device resolution can fail if the device is not monitored with NDM, or the LLDP or CDP data is insufficient to match the discovered device to the monitored device.
```

Comment:
It seems like these requirements are mentioned multiple times in the topic. Consider just mentioning them in the Prereq's and linking to that. Just easier than repeating it, IMO.